modules/RStudio/RStudio.Rmd

```diff
@@ -50,6 +52,7 @@ knitr::include_graphics("https://ayeimanolr.files.wordpress.com/2013/04/r-rstudi
 <!-- ## RStudio/R Console ![Console](images/r-rstudio-1-3-console.png) -->
 
 ## Getting the editor
+You must open a to save code to open the editor. These include for example R Scripts or R Markdown.
```

Comment:
You must open what? A script ?

Reply:
something got deleted :(  I think it was the word file.

Reply:
fixed now adding the word file